dockers/tpu-tests/tpu_test_cases.jsonnet

```diff
@@ -8,7 +8,7 @@ local tputests = base.BaseTest {
   mode: 'postsubmit',
   configMaps: [],
 
-  timeout: 900, # 15 minutes, in seconds.
+  timeout: 3000, # 50 minutes, in seconds.
```

Comment:
Do we really need 50 minutes? Or did you increase it to debug?

Keep it as small as possible, to avoid unnecessary waiting when something hangs.

Reply:
it is upper bound so hope to get faster but shall not exceed 50min
also, do we still have the other corn job about killing any long-running TPU jobs?

Reply:
I'll tighten it a bit once the current failures are resolved.